data/mz/health/hcm-admin-console/facilitySchema.json

```diff
@@ -38,8 +37,7 @@
                 "HCM_ADMIN_CONSOLE_FACILITY_NAME",
                 "HCM_ADMIN_CONSOLE_FACILITY_TYPE",
                 "HCM_ADMIN_CONSOLE_FACILITY_STATUS",
-                "HCM_ADMIN_CONSOLE_FACILITY_CAPACITY",
-                "HCM_ADMIN_CONSOLE_BOUNDARY_CODE"
```

Comment:
its removed from required because while everywhere we dont need boundary code like which has already facility code it is only for search ...... if it is not for search if it is for create then boundary code will be validated internally